concepticondata/references/references.bib

```diff
@@ -4113,3 +4113,12 @@ @Article{Oskolskaya2021
   author                              = {Sofia Oskolskaya and Ezequiel Koile and Martine Robbeets},
   title                               = {A Bayesian approach to the classification of Tungusic languages}
 }
+
+@Article{Gerardi2021c,
```

Comment:
Volume and Number and Page range are given in the PDF we prepared of your blog post.